apps/web/src/pages/Users/AdminApplicantProfilePage/AdminApplicantProfilePage.tsx

```diff
@@ -56,51 +62,147 @@ interface AdminApplicantProfileProps {
+              <TableOfContents.AnchorLink id="basic-information">
+                {intl.formatMessage({
+                  defaultMessage: "Basic information",
+                  id: "RDFAWE",
+                  description: "Title for basic information",
+                })}
+              </TableOfContents.AnchorLink>
+              <TableOfContents.List className="mt-1.5">
+                <TableOfContents.ListItem>
+                  <TableOfContents.AnchorLink id={PERSONAL_CONTACT_INFO_ID}>
+                    {intl.formatMessage(
+                      profileMessages.personalAndContactInformation,
+                    )}
+                  </TableOfContents.AnchorLink>
+                </TableOfContents.ListItem>
+                <TableOfContents.ListItem>
+                  <TableOfContents.AnchorLink id={LANGUAGE_PROFILE_ID}>
+                    {intl.formatMessage(profileMessages.languageProfile)}
+                  </TableOfContents.AnchorLink>
+                </TableOfContents.ListItem>
+                <TableOfContents.ListItem>
+                  <TableOfContents.AnchorLink id={WORK_PREFERENCES_ID}>
+                    {intl.formatMessage(navigationMessages.workPreferences)}
+                  </TableOfContents.AnchorLink>
+                </TableOfContents.ListItem>
+                <TableOfContents.ListItem>
+                  <TableOfContents.AnchorLink id={DEI_ID}>
+                    {intl.formatMessage(
+                      navigationMessages.diversityEquityInclusion,
+                    )}
+                  </TableOfContents.AnchorLink>
+                </TableOfContents.ListItem>
+                <TableOfContents.ListItem>
+                  <TableOfContents.AnchorLink id={GOV_INFO_ID}>
+                    {intl.formatMessage(profileMessages.govEmployeeInformation)}
+                  </TableOfContents.AnchorLink>
+                </TableOfContents.ListItem>
+              </TableOfContents.List>
+            </TableOfContents.ListItem>
+          </TableOfContents.List>
+          <Separator orientation="horizontal" space="xs" decorative />
+          <DownloadButton id={user.id} />
+        </TableOfContents.Navigation>
+        <TableOfContents.Content>
+          <div className="mb-6 flex flex-col justify-between gap-3 sm:flex-row sm:items-end">
+            <Heading icon={UserCircleIcon} color="secondary" className="m-0">
+              {intl.formatMessage({
+                defaultMessage: "Basic information",
+                id: "RDFAWE",
+                description: "Title for basic information",
+              })}
+            </Heading>
+            <div className="flex items-end gap-3">
+              <Button mode="inline" color="primary" onClick={toggleSections}>
+                {hasOpenSections
+                  ? intl.formatMessage({
+                      defaultMessage:
+                        "Collapse all <hidden>application information</hidden>sections",
```

Comment:
I think this should be like this:
`Collapse all<hidden> application information</hidden> sections`